components/brave_webtorrent/extension/components/torrentViewer.tsx

```diff
@@ -6,7 +6,6 @@ import * as React from 'react'
 
 // Assets
 require('../../../../ui/webui/resources/fonts/muli.css')
-require('../styles/styles.css')
```

Comment:
Hey, I don't know that removing the stylesheet here is the right solution. I'm not too familiar with this area of the codebase, but it looks like it has a bunch of default styles to display images/video and style a few other things. Is it all unused?

Reply:
There is absolutely no difference to the torrents page when removing the style sheet. I think it's just not loaded properly as I initially tried to solve this issue with changes there.

Reply:
Ah that's weird as - maybe the styles not being applied is the root cause of this?

Reply:
That is quite likely. However, I couldn't figure out why the styles weren't applied so I made this change.